evap/staff/views.py

```diff
@@ -410,7 +410,7 @@ def semester_raw_export(request, semester_id):
         _('#Participants'), _('#Comments'), _('Average grade')])
     for course in semester.course_set.all():
         degrees = ", ".join([degree.name for degree in course.degrees.all()])
-        course.avg_grade, course.avg_deviation = calculate_average_grades_and_deviation(course)
+        course.avg_grade, course.avg_deviation, __ = calculate_average_grades_and_deviation_and_distribution(course)
```

Comment:
the distribution should not go into the raw export?

Reply:
i didn't think about that yet. what would you say?

Reply:
i don't know! i'm not the user :) i actually have no idea right now how the raw export is used at all.

Reply:
well... the user would be me i guess. the raw data export is used to create additional statistics - what we want to find changes every time, that's why the "raw" data is a good way to just get some numbers to fiddle around with.
but currently i can't think of a use case where we would need the distribution in there.